src/internal/connector/exchange/transform.go

```diff
@@ -292,6 +297,19 @@ func sanitizeEvent(orig models.Eventable) (models.Eventable, error) {
 	return newEvent, nil
 }
 
+func isNil(i interface{}) bool {
```

Comment:
use `any` in place of `interface{}`
```suggestion
func isNil(i any) bool {
```

Reply:
nope.  IMO, you should add this to `ptr` for widespread usage.